app/views/static_pages/home.html.erb

```diff
@@ -1,28 +1,111 @@
-<div class="w-full h-full">
-	<h1>home page</h1>
+<div class="w-full h-full max-w-screen-2xl mx-auto px-7 md:px-24">
+	<div class="min-h-screen relative flex justify-center items-center">
+		<div class="w-full flex flex-col justify-center items-center lg:flex-row lg:justify-between">
+			<div class="lg:w-6/12 text-center lg:text-left order-2 lg:order-1 relative">
+				<div class="lg:absolute lg:-top-56 z-20">
+					<p class="text-xs text-slate-500 uppercase font-medium">3D-Друк</p>
+					<h1 class="text-3xl lg:text-6xl text-gray-800 pt-2 font-medium">Твоя ідея,<br /> наше втілення</h1>
+				</div>
+			</div>
+			<div class="w-full lg:w-4/12 order-1 lg:order-2">
+				<div class="w-full lg:w-4/5 lg:absolute top-6 -right-7 z-10">
+					<%= image_tag("home_img/models.png") %>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="relative flex flex-col justify-center items-center">
+		<h4 class="absolute bottom-14 text-center text-zinc-500 uppercase font-medium pb-2 bg-stone-200 z-20">Гортай вниз
+		</h4>
+		<div class="absolute bottom-4 z-10">
+			<%= image_tag("home_img/arrow.png") %>
+		</div>
+	</div>
+	<div class="w-full px-4 lg:px-0 lg:w-2/4 mx-auto py-20 text-zinc-500 text-center">
+		<p>Друк здійснюється спеціальним пристроєм — 3D-принтером, який забезпечує створення фізичного об'єкта шляхом
```

Comment:
plz. move Ukrainian text in uk.locale file 
And change this text => t.locale
@AndriiBuhalo will help you

Reply:
done